knowledge-base/kb-filedialogsdemos-notavailable.md

```diff
@@ -0,0 +1,37 @@
+---
+title: FileDialogs Demos Not Available in WPF Control Examples App
+description: "How to run FileDialogs WPF Demos application." 
```

Comment:
This is a troubleshooting article so use a more appropriate description here  that is not starting with "How to"